requirements.txt

```diff
@@ -2,7 +2,7 @@ arrow
 beautifulsoup4
 bibtexparser
 deprecated
-fake_useragent
+fake_useragent>=1.0
```

Comment:
This shouldn't be required for any new installation, but good to have this anyway.